Modern/utilities/src/test/java/org/bonej/utilities/SharedTableTest.java

```diff
@@ -70,17 +70,17 @@ public void testAdd() {
 		SharedTable.add(labelA, header2, 3.0);
 
 		// VERIFY
-		final Table<DefaultColumn<String>, String> table = SharedTable.getTable();
+		final Table<DefaultColumn<Double>, Double> table = SharedTable.getTable();
 		assertEquals("Wrong number of columns", 2, table.getColumnCount());
 		assertEquals("Wrong number of rows", 3, table.getRowCount());
-		final DefaultColumn<String> column1 = table.get(header1);
+		final DefaultColumn<Double> column1 = table.get(header1);
 		assertEquals("Cell should be empty", EMPTY_CELL, column1.get(0));
 		assertEquals("Wrong number of empty cells", 1, column1.stream().filter(
-			s -> s.equals(EMPTY_CELL)).count());
-		final DefaultColumn<String> column2 = table.get(header2);
-		assertEquals("Cell contains wrong value", "3.0", column2.get(1));
+			s -> s == null).count());
+		final DefaultColumn<Double> column2 = table.get(header2);
+		assertEquals("Cell contains wrong value", new Double(3.0), column2.get(1));
```

Comment:
Could use Double.getDoubleValue() instead of boxing the literal

Reply:
As above, assertEquals(String, double, double) is deprecated.